modules/Dockerfile

```diff
@@ -15,33 +16,45 @@ FROM "${DOCKER_HUB_PROXY}python:3.9-slim-bullseye" as python-build
         libpoppler-cpp-dev \
         libfuzzy-dev \
         libssl-dev \
+        ninja-build \
         && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
     
     RUN mkdir /wheels
     WORKDIR /srv
 
-    RUN if [ ! -z ${MODULES_COMMIT} ]; then \
-        git clone https://github.com/MISP/misp-modules.git /srv/misp-modules && cd /srv/misp-modules && git checkout ${MODULES_COMMIT}; \
-        else git clone --branch ${MODULES_TAG} --depth 1 https://github.com/MISP/misp-modules.git /srv/misp-modules; fi
+    RUN <<-EOF
+            if [ ! -z ${MODULES_COMMIT} ]; then
+                git clone https://github.com/MISP/misp-modules.git /srv/misp-modules && cd /srv/misp-modules && git checkout ${MODULES_COMMIT}
+            else
+                git clone --branch ${MODULES_TAG} --depth 1 https://github.com/MISP/misp-modules.git /srv/misp-modules
+            fi
 
-    RUN cd /srv/misp-modules; \
-        echo "pyeti" >> REQUIREMENTS; \
-        echo "greynoise" >> REQUIREMENTS; \
-        echo "git+https://github.com/abenassi/Google-Search-API" >> REQUIREMENTS; \
-        sed -i 's|apiosintds.*|git+https://github.com/davidonzo/apiosintDS\@misp|g' REQUIREMENTS; \
-        sed -i 's/-e //g' REQUIREMENTS; \
-        pip3 wheel -r REQUIREMENTS --no-cache-dir -w /wheels/
+            cd /srv/misp-modules
+            echo "pyeti" >> REQUIREMENTS
+            echo "greynoise" >> REQUIREMENTS
+            echo "git+https://github.com/abenassi/Google-Search-API" >> REQUIREMENTS
+            sed -i 's|.*apiosintDS.*|apiosintDS==2.0.1|g' REQUIREMENTS
+            # let apiosintDS decide since misp-modules doesn't actually use this package anyway
+            sed -i '/validators.*/d' REQUIREMENTS
+            sed -i 's/-e //g' REQUIREMENTS
+            pip3 wheel -r REQUIREMENTS --no-cache-dir -w /wheels/
+            rm -rf /srv/misp-modules
+EOF
 
-    RUN git clone --depth 1 https://github.com/stricaud/faup.git /srv/faup; \
-        cd /srv/faup/build || exit; cmake .. && make install; \
-        cd /srv/faup/src/lib/bindings/python || exit; pip3 wheel --no-cache-dir -w /wheels/ .
+    RUN <<-EOF
+        git clone --depth 1 https://github.com/stricaud/faup.git /srv/faup
+        cd /srv/faup
+        git checkout ${LIBFAUP_COMMIT}
```

Comment:
Shall we make this conditional iff ${LIBFAUP_COMMIT} is non-empty?

I am getting concerned about existing pipelines not setting it in the environment file